javascript/QUIZ APP/javascript.js

```diff
@@ -0,0 +1,128 @@
+const quizQuestion = [
+  {
+    id: 1,
+    question: "What is the chemical symbol for water?",
+    options: [
+      { text: "CO2", isCorrect: false },
+      { text: "H2O", isCorrect: true },
+      { text: "O2", isCorrect: false },
+      { text: "CH4", isCorrect: false },
+    ],
+    explanation: "H2O is the chemical symbol for water",
+  },
+  {
+    id: 2,
+    question: "Who wrote the play “Romeo and Juliet”?",
+    options: [
+      { text: "William Shakespeare", isCorrect: true },
+      { text: "Jane Austen", isCorrect: false },
+      { text: "Charles Dickens", isCorrect: false },
+      { text: "Mark Twain", isCorrect: false },
+    ],
+    explanation: "William Shakespeare wrote the play “Romeo and Juliet”",
+  },
+  {
+    id: 3,
+    question: "Which planet is known as the “Red Planet”?",
+    options: [
+      { text: "Mars", isCorrect: true },
+      { text: "Venus", isCorrect: false },
+      { text: "Jupiter", isCorrect: false },
+      { text: "Saturn", isCorrect: false },
+    ],
+    explanation: "Mars is known as the “Red Planet”.",
+  },
+  {
+    id: 4,
+    question: "What is the currency of Denmark?",
+    options: [
+      { text: "Kroner", isCorrect: true },
+      { text: "Pound", isCorrect: false },
+      { text: "Dollar", isCorrect: false },
+      { text: "Rupees", isCorrect: false },
+    ],
+    explanation: "Kroner is the currency of Denmark.",
+  },
+];
+
+// Creating form and styling the form
+const form = document.createElement("form");
+document.body.appendChild(form);
+form.style.alignItems = "center";
+form.style.margin = "2%";
```

Comment:
Giving margin a % isn't the best practice. If you want to create a responsive design then I would recommend look at the [media queries](https://www.w3schools.com/css/css3_mediaqueries.asp).